aerich/migrate.py

```diff
@@ -42,8 +42,7 @@ class Migrate:
     _upgrade_m2m: List[str] = []
     _downgrade_m2m: List[str] = []
     _aerich = Aerich.__name__
-    _rename_old: List[str] = []
-    _rename_new: List[str] = []
+    _rename_fields: Dict[str, Dict[str, str]] = {}  # {'model': {'old_field': 'new_field'}}
```

Comment:
Sorry, I haven't used aerich much. Can you clarify how does aerich handles field renaming? Does it ask you to confirm that the field is renamed? I know Django asks you to confirm that because it's kinda hard to distinguish two operations:
- field deleted and added
- field renamed

Reply:
@henadzit Thanks for you advice. Description/comment/document is importance, get it~

Reply:
Thank you, the description helped to understand the issue and solution!